homeassistant/components/probe_plus/icons.json

```diff
@@ -0,0 +1,24 @@
+{
+  "entity": {
+    "sensor": {
+      "probe_temperature": {
+        "default": "mdi:thermometer-bluetooth"
+      },
+      "probe_battery": {
+        "default": "mdi:battery"
+      },
+      "relay_battery": {
+        "default": "mdi:battery"
+      },
+      "probe_rssi": {
+        "default": "mdi:bluetooth-connect"
+      },
+      "probe_voltage": {
+        "default": "mdi:flash-triangle"
+      },
+      "relay_voltage": {
+        "default": "mdi:flash-triangle"
+      }
```

Comment:
Wondering if these can also just use the device class icon

Reply:
will test and see what happens